### PR TITLE
This is a relatively minor change to Saltern-2 intended to become a series of updates to the famed map

### PR DESCRIPTION
**Pull request by Sirius-Frost**
## About the PR
 
The evacuation center now has a medical station and has generally been refurbished and revamped.

## Why / Balance

This would make evacuating spessmen in bad or critical condition much easier as they can be 'easily' healed when shit hits the fan and accessing the rest of the station, especially medical, becomes difficult if not impossible. I made it this way not only for the aforementioned quality of life feature, but to overall fit the theme of an evacuation center better and make it generally more pleasing to the eye. 
 
## Technical details

There are no code changes.

## Media

Second
First

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

None

**Changelog**

:cl:
- add: Medical station to evac w/ minimal supplies and a fun note
- remove: removed Mr. Chang vendor from just below the Chapel
- remove: Removed the chairs in evac. No sitting!
- tweak: Changed tiling in evac
- tweak: Replaced oxygen cabinet and fire cabinet above disposal in evac with a fire extinguisher
- tweak: Replaced vending machine near east hallway to evac with a more themed one with its adjacent poster
- tweak: Replaced all lights in evac with LEDs
- tweak: Replaced signs on the evac docking bays with more appropriately placed/themed ones.
- tweak: Moved the fire extinguisher in the arrivals to evac doorway further into arrivals to prevent having to open the door to access it.

### CODE REVIEW

**Review by Merrokitsune — CHANGES_REQUESTED** [review]
Thank you very much for your first pull-request.

Unfortunately, I spotted some things that don’t seem to be right. Besides, if I may ask nicely, are you trying to update Saltern-2. If not, do you want the map to be put into rotation or not? Just wanted to clear things up.

**Comment by Sirius-Frost** [comment]
****

> Thank you very much for your first pull-request.
> 
> Unfortunately, I spotted some things that don’t seem to be right. Besides, if I may ask nicely, are you trying to update Saltern-2. If not, do you want the map to be put into rotation or not? Just wanted to clear things up.

Could you specify what doesn't seem right and how I could go about fixing them? 

Ideally, this would in-fact be an update to Saltern-2. I have named it Saltern-3 for ease of cross-referencing while mapping and testing by myself. I'm not sure it would make sense to put it into rotation by itself in its current form, but given my plans to radically alter certain parts of the layout such as the location of the Observatory it might be wiser to plan for it to be its own, separate map. I'd very much appreciate feedback in that regard.

**Comment by Merrokitsune** [comment]
> Could you specify what doesn't seem right and how I could go about fixing them?

The saltern-3.yml prototype has a misplaced path.

**Review by Merrokitsune — APPROVED** [review]
Sounds good. Now, we’ll see what to do, due to the unique nature of this PR (you’re intent on adding another Saltern that is already in rotation to be put into rotation).

File: Resources/Prototypes/Ronstation/Maps/saltern-3.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+- type: gameMap
+  id: Saltern-3
+  mapName: 'Saltern'
+  mapPath: C:\Users\SiriusFrost\OneDrive\Desktop\ronstation\Resources\Maps\Ronstation\saltern-3.yml
```
[the remark this answers is not in the document]
Comment (Merrokitsune) [reply Resources/Prototypes/Ronstation/Maps/saltern-3.yml]: Resolved by commit 45713e7819117ec20df6b02cf0c558a17f0a93fa.

**Comment by Sirius-Frost** [comment]
> > …completely move the observatory over to arrivals which would radically change Saltern.
> 
> It would be nice if there was a screenshot for that, so we can see what it is. ;) Anyways, what would the previous location for the obsevatory be?

I haven't done anything so radical yet. This is just a test run so its only what I put in the initial PR that got changed. I was talking to Sad earlier about what could be some potential revisions to the observatory when i get to it.

I felt like putting everything into one PR would make for an all or nothing scenario and I didnt want to spend hours doing that incase the PR didnt get merged for one reason or the other. As for the previous location for the observatory - its still an offshoot of maints below engineering.  its in a VERY peculiar position on the map.

just fixed the mapPath for the second time as well. used the wrong punctuation.

**Comment by Sirius-Frost** [comment]
> I am terribly sorry for having to request changes again, but it appears that I have been wrong in the map path.
Could you show me what its actually supposed to be again? I tried fixing it but I got it wrong again seemingly.

**Comment by Merrokitsune** [comment]
> > I am terribly sorry for having to request changes again, but it appears that I have been wrong in the map path.
> 
> Could you show me what its actually supposed to be again? I tried fixing it but I got it wrong again seemingly.

It is supposed to be “/Maps/Ronstation/saltern-3.yml”.

Just a little question, are you able to view suggested changes in my reviews? My reviews should be viewable.
IMG_6009

**Comment by Sirius-Frost** [comment]
> > > I am terribly sorry for having to request changes again, but it appears that I have been wrong in the map path.
> > 
> > 
> > Could you show me what its actually supposed to be again? I tried fixing it but I got it wrong again seemingly.
> 
> It is supposed to be “/Maps/Ronstation/saltern-3.yml”. Take the path from Saltern-2’s prototype as reference. IMG_6008
> 
> Just a little question, are you able to view suggested changes in my reviews, since I made suggested changes for you to follow? My reviews, as well as the suggested changes, should be viewable. IMG_6009

I can see that you requested changes, but i usually cant see the specific review for some reason. i only saw the one you posted at 40 minutes ago

**Comment by Sirius-Frost** [comment]
```
Failed!  - Failed:     1, Passed:   367, Skipped:     0, Total:   368, Duration: 7 m 23 s - Content.IntegrationTests.dll (net8.0)
Error: Process completed with exit code 1.
```

Oh, cmon.

**Comment by Merrokitsune** [comment]
> ```
> Failed!  - Failed:     1, Passed:   367, Skipped:     0, Total:   368, Duration: 7 m 23 s - Content.IntegrationTests.dll (net8.0)
> Error: Process completed with exit code 1.
> ```
> 
> Oh, cmon.

It’s okay.
We’re going to discuss what to do with this PR, so please await orders.
However, if you are going to fix this error…
- go to “`Content.Benchmarks/MapLoadBenchmark.cs`” and add “`Saltern-3`” to the “`public static readonly string[] MapsSource = {}`” (should be located in line 43).
- go to }`Content.IntegrationTests/Tests/PostMapInitTest.cs`” and add “`Saltern-3`” to the “`private static readonly string[] GameMaps = {}`” list (should be located in line 46).

**Comment by Sirius-Frost** [comment]
> > ```
> > Failed!  - Failed:     1, Passed:   367, Skipped:     0, Total:   368, Duration: 7 m 23 s - Content.IntegrationTests.dll (net8.0)
> > Error: Process completed with exit code 1.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Oh, cmon.
> 
> It’s okay. We’re going to discuss what to do with this PR, so please await orders. However, if you are going to fix this error…
> 
> * go to “`Content.Benchmarks/MapLoadBenchmark.cs`” and add “`Saltern-3`” to the “`public static readonly string[] MapsSource = {}`” (should be located in line 43).
> * go to }`Content.IntegrationTests/Tests/PostMapInitTest.cs`” and add “`Saltern-3`” to the “`private static readonly string[] GameMaps = {}`” list (should be located in line 46).

I've done as suggested. Hopefully that fixes the error as you've said. I'll await the councils judgement.

**Review by Merrokitsune — APPROVED** [review]
Forgot to approve, sorry.

**Review by Merrokitsune — CHANGES_REQUESTED** [review]
Hello again, sirius_frost.

After some discussion with staff and community members, we decided that the following are decided:

This should be kept as a general saltern-2 update, not to be a new map to be put into rotation. Why do we need 2 identical maps when you can have 1?

medical bay in evacuation should be replaced with something else, or otherwise removed, for the following reasons:
- Medbay in evac would be abuseable by players, who would simply break in and take the meds stored within that room, even the locked nature of these storage items aren’t enough of a deterrent to put a stop to these activities. At least, medical personnel can bring their medicine to evacuation if needed.
- Evac medbay is already close to the main medbay area (like it takes around 10 seconds to travel from evac medbay to main medbay), so even medical personnel might find it redundant.

As for the replacement, I would recommend something like a help desk with at least 1 first aid kit.

**Comment by Sirius-Frost** [comment]
> Hello again, sirius_frost.
> 
> After some discussion with staff and community members, we decided that the following are decided:
> 
> This should be kept as a general saltern-2 update, not to be a new map to be put into rotation. Why do we need 2 identical maps when you can have 1?
> 
> medical bay in evacuation should be replaced with something else, or otherwise removed, for the following reasons:
> 
> * Medbay in evac would be abuseable by players, who would simply break in and take the meds stored within that room, even the locked nature of these storage items aren’t enough of a deterrent to put a stop to these activities. At least, medical personnel can bring their medicine to evacuation if needed.
> * Evac medbay is already close to the main medbay area (like it takes around 10 seconds to travel from evac medbay to main medbay), so even medical personnel might find it redundant.
> 
> As for the replacement, I would recommend something like a help desk with at least 1 first aid kit.

Understood. In the screenshots, the only things in the crates are MREs and bodybags. There are no medkits stored there - the paper stamped with the centcomm stamp notifies the player as much. the one medicinal locker and defib is all they have already, and those could be removed too.

I understand if its too close to medbay to warrant it. i was thinking about if there were no power and somebody didnt have a crowbar - itd take far longer to get to the actual medbay. thats why its in evac - to act as emergency medical attention. the idea is that throughout the shift, medical team works to stock it in case of emergency - when youre hanging out in evac you can go ahead and patch everyone up best you can before the shuttle arrives.

as for making a help desk? maybe. albeit i dont see much point having that in evac. id probably just revert it to where theres no desk and you have the setup with all the chairs. maybe make a help desk in arrivals that leads into the dormitories to funnel people that way rather than through evac (which I always found a little odd)

**Review by Merrokitsune — CHANGES_REQUESTED** [review]
I found a missing coma in line 67 of Content.IntegrationTests/Tests/PostMapInitTest.cs.

Also, I hope I'm not wrong, but...
> This should be kept as a general saltern-2 update, not to be a new map to be put into rotation. Why do we need 2 identical maps when you can have 1?

- https://github.com/RonRonstation/ronstation/pull/148#pullrequestreview-2468715474

So, if you insist on having Saltern-3, I can derotate (and eventually remove) Saltern-2.

File: Content.IntegrationTests/Tests/PostMapInitTest.cs [hunk 2]
```diff
@@ -65,6 +65,7 @@ public sealed class PostMapInitTest
             "Saltern2",
             "Cluster2",
             "Atlas2"
```
Comment (Merrokitsune) [inline Content.IntegrationTests/Tests/PostMapInitTest.cs]: This is the source of the errors.
```suggestion
            "Atlas2",
```

**Review by Merrokitsune — CHANGES_REQUESTED** [review]
Hello, Sirius_Frost.
I have finally checked the map and noticed 3 problems.
1. An amount of space within the evac medbay is empty and lacks air. Perhaps, you forgot to type `fixgridatmos <gridid>` into the console?
2. The directional windows outside the main windows lack tiles to support and will float aimlessly. Besides, it may obstruct the evacuation shuttle from being near.
3. The fact that the window is facing space without any protections might spell disaster and this are may be spaced more often than you can imagine. So, I can recommend either moving the windows back and adding a protection liner, or converting the windows to reinforced walls.

I have an image proof for these. 
Screenshot from 2024-12-01 12-39-05

On a positive note, the map is up-to-date with the latest Saltern-2.

**Review by Merrokitsune — APPROVED** [review]
This can work.

**Comment by Sirius-Frost** [comment]
I absolutely fucked the push to origin files. attempting to fix.

**Review by Merrokitsune — APPROVED** [review]
It should work now, merging.

**Comment by Merrokitsune** [comment]
You can delete the branch now.

**Pull request merged**